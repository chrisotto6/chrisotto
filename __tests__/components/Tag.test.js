import { render, fireEvent } from '@testing-library/react'
import Tag from '@/components/Tag'

it('renders Tag unchanged', () => {
  const { container } = render(<Tag text="test" />)
  expect(container).toMatchSnapshot()
})
