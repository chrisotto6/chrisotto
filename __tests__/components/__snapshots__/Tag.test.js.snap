// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tag unchanged 1`] = `
<div>
  <a
    class="mr-3 text-sm font-medium uppercase text-primary-500 hover:text-primary-600 dark:hover:text-primary-400"
    href="/tags/test"
  >
    test
  </a>
</div>
`;
